uuu/uuu.cpp

```diff
@@ -323,9 +329,10 @@ string build_process_bar(size_t width, size_t pos, size_t total)
 
 void print_auto_scroll(string str, size_t len, size_t start)
 {
+    MAX_PROGRESS_WIDTH = max(str.size(), MAX_PROGRESS_WIDTH);
 	if (str.size() <= len)
```

Comment:
how about if two boards connected?
1: 8   FB: [===>         ] some information
1: 10 FB: [===>         ] some information xxx

second line will impact the first line, because use global variable.

Reply:
Certainly, when `print_auto_scroll` is reached while rendering second line, `MAX_PROGRESS_WIDTH` will be set based on longest progress status " some information xxx"

First line would then be printed as follow:
```
1: 8 FB: [===> ] some information     \EOL
1: 10 FB: [===> ] some information xxx\EOL
```

Reply:
MAX_PROGRESS_WIDTH look like a macro, it is should be g_max_process_width. 
But I still have problem,  MAX_PROGRESS_WIDTH = max(str.size(), MAX_PROGRESS_WIDTH); MAX_PROGRESS_WIDTH  may be longer than console width, 

str.resize(MAX_PROGRESS_WIDTH, ' ');
cout << str;

it may cause new line
```
      1: 8 FB: [===> ] some information     \EOL
      1: 10 FB: [===> ] some information 
      xxx\EOL
```